app/routines/search_users.rb

```diff
@@ -36,33 +40,51 @@ class SearchUsers
   SORT_DESCENDING = 'DESC'
   MAX_MATCHING_USERS = 10
 
+  def matches(field)
+    debugger
```

Comment:
Opps I did indeed.  Actually the whole method needs removed.  That was an earlier thing I was noodling on